resource/_imports.pyi.em

```diff
@@ -3,10 +3,21 @@
 third_party_imports = set()
 first_party_imports = set()
 generator(component, defined_classes, third_party_imports, first_party_imports)
+
+import sys
+python_minor_version = sys.version_info.minor
 }@
+
 import typing
+@[if python_minor_version < 10]@
+from typing_extensions import TypeAlias
+@[else]@
+from typing import TypeAlias
+@[end if]
```

Comment:
```suggestion

import sys
import typing

if sys.version_info < (3, 10):
    from typing_extensions import TypeAlias
else:
    from typing import TypeAlias
```

- IMHO, when checking a Python version, it would be better to compare with a tuple including the major version.
- mypy recognize the use of `sys.version_info` when static-type checking. As such, it would be nice to put the condition to stub files as well to increase the portability of the generated stub files.
  - https://mypy.readthedocs.io/en/stable/common_issues.html#python-version-and-system-platform-checks

Reply:
Oh that's a great point! I was looking into this, but I had read that mypy ignored conditional imports... I should have read more deeply the documentation!